yaptide/admin/db_manage.py

```diff
@@ -68,7 +69,7 @@ def list_users(**kwargs):
     ResultSet = ResultProxy.fetchall()
     click.echo(f"{len(ResultSet)} users in DB:")
     for row in ResultSet:
-        click.echo(f"Login {row.username} ; Passw ...{row.password_hash[-10:]} ; Proxy {proxy_hash(row.grid_proxy)}")
+        click.echo(f"Login {row.username}; Passw ...{row.password_hash[-10:]}; Proxy {proxy_hash(row.grid_proxy)}")
```

Comment:
Get rid of proxy hash or replace it with key hash

Reply:
ok